vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -535,6 +535,15 @@ bool FrameworkElementViewManager::UpdateProperty(
     } else if (propertyName == "accessibilityActions") {
       auto value = json_type_traits<winrt::IVector<winrt::react::uwp::AccessibilityAction>>::parseJson(propertyValue);
       DynamicAutomationProperties::SetAccessibilityActions(element, value);
+    } else if (propertyName == "display") {
+      if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::String) {
+        auto value = propertyValue.AsString();
+        if (value == "none") {
+          element.Visibility(winrt::Windows::UI::Xaml::Visibility::Collapsed);
```

Comment:
Check out our namespace aliases - the winrt:ui:xaml bit should be `xaml` 
Look in CPPWinrtIncludes.h@16

Reply:
It's currently failing the WINUI3 build in the CI